pkg/controllers/certificaterequest_controller_test.go

```diff
@@ -95,9 +95,11 @@ func TestCertificateRequestReconcile(t *testing.T) {
 						Namespace: "ns1",
 					},
 					Spec: issuerapi.AWSPCAIssuerSpec{
-						SecretRef: v1.SecretReference{
-							Name:      "issuer1-credentials",
-							Namespace: "ns1",
+						SecretRef: issuerapi.SecretReference{
+							SecretReference: v1.SecretReference{
```

Comment:
This... feels redundant? Can you set `Name` and `Namespace` on `SecretRef` itself?

Reply:
I don't think there's a way around this when using composition. Since it's only making the tests a bit uglier it seems acceptable to me.

If there's another method you can point me to, I can update.

Reply:
Oh yeah now it's just the Go code being uglier, the CRDs don't have this problem anymore (thanks to the JSON change in a comment above). The CRDs give a poor user experience, but the go code is internal so I don't really mind. Cool!